tests/migration_test.go

```diff
@@ -296,6 +296,15 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 		}
 	}
 
+	// temporaryNodeDrain also sets the `NoSchedule` taint on the node.
+	// nodes with this taint will be reset to their original state on each
+	// test teardown by the test framework. Check libnode.CleanNodes,
+	// https://github.com/kubevirt/kubevirt/blob/142491b5e78ce66cf5b8bfe386129618fdc47b70/tests/libnode/node.go#L55
+	temporaryNodeDrain := func(nodeName string) {
```

Comment:
Can we create a regular function instead please ?

I think that since the code changes, no need the URL (even that this is permanent link, main code might change and main code is what matters)

Reply:
I could make it a regular function, but to do so, I'd have to also move the drain func and make that a regular func. I wanted to keep changes to a minimum.

If "code changes", I would say part of those changes should be to remove the then-outdated comment.

Right now, I think indicating that the node cleanup **does not** happen unless the node is tainted is important enough to deserve a comment, since that isn't explicitly indicated anywhere - that I'm aware of.

Reply:
> If "code changes", I would say part of those changes should be to remove the then-outdated comment.
> 
> Right now, I think indicating that the node cleanup **does not** happen unless the node is tainted is important enough to deserve a comment, since that isn't explicitly indicated anywhere - that I'm aware of.

Talking only about the URL imo, rest can stay for now

Reply:
The URL just points to the code I've stated before - it is meant to help the reader find out the info I'm advertising.

If you think that's redundant (I do not), I can oblige and remove the URL.

Reply:
imho better to remove the URL, thanks